.github/workflows/dask_runner_tests.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
why windows is removed?

Reply:
Ah, something was lost in the shuffle: https://github.com/apache/beam/pull/27618/files#r1271370322

IIRC writing this so long ago, a dependency (with dask?) caused windows to fail. However, this may have “resolved itself” given how much the code has changed.